### PR TITLE
Update dependency versions and fix project.json schema warnings

### PR DESCRIPTION
**Pull request by pranavkm**

### CODE REVIEW

**Comment by pranavkm** [comment]
cc @mikeharder \ @ajaybhargavb

**Comment by mikeharder** [comment]
:shipit:

File: src/Benchmarks/project.json [hunk 1]
```diff
@@ -1,52 +1,47 @@
...
-      "version": "1.0.0-*",
-      "imports": [
-        "dnxcore50",
-        "portable-net452+win81"
-      ]
-    }
+    "Microsoft.EntityFrameworkCore.Tools": "1.0.0-*"
   },
-  "content": [
-    "appsettings.json",
```
Comment (ajaybhargavb) [inline src/Benchmarks/project.json]: Were these meant to be copied only to build output? Otherwise need to add them to `publishOptions`.

**Comment by ajaybhargavb** [comment]
:shipit:

File: src/Benchmarks.ClientJob/project.json [hunk 2]
```diff
@@ -1,26 +1,24 @@
 {
-  "version": "1.0.0-*",
+  "version": "1.1.0-*",
   "description": "Class to transfer data to/from the benchmark client.",
-  "repository": {
-    "type": "git",
-    "url": "git://github.com/AspNet/Benchmarks"
+  "packOptions": {
```
Comment (GuardRex) [inline src/Benchmarks.ClientJob/project.json]: I couldn't get `repository` to work either way ... at the root level or the `packOptions` level. I opened an issue to ask about it: https://github.com/dotnet/cli/issues/3823

Comment (pranavkm) [inline src/Benchmarks.ClientJob/project.json]: No idea, it's the only place it doesn't complain though.

**Pull request closed without merging**